### ISSUE TITLE
Date format without Z

### ISSUE THREAD

**Issue by danielf-rodriguez** [loopbackio/strong-soap#289]
Is there a way to make date format on XML request comes without the Z at the end?. We have a legacy web service that is validating the format to be YYYY-MM-DD and the Z is causing problems.

**Comment by Scrib3r** [comment]
Yes, It's a big problem for us too. A soap server expects date in format YYYY-MM-DD but in every requests z symbol added into the end of field of date. Can anyone help us?

**Comment by kellyjandrews** [comment]
Going to have to reopen this one - it's changing all dates into UTC Zulu time - I'm honestly not sure how. I specifically need these to be in PST.  Is there a way to do this?

**Comment by kellyjandrews** [comment]
So in my case, the API doesn't care about the time zone, so using `moment().subtract(7, 'hours')` fixed it.

**Comment by hrushikesh-udchalo** [comment]
Has anybody found , How to solve this issue ?

**Comment by pkfms** [comment]
Hi, I also need to send dates to an api that requires dates in format 2022-02-08T10:27:18+01:00 instead of Z. Would be great to be able to configure this in the request

**Comment by Scrib3r** [comment]
Hello. I've created MR for extending this issues https://github.com/loopbackio/strong-soap/pull/525
